frappe/erpnext
# fix: Get MRs that are yet to be received but fully ordered in Report `Requested Items to Order and Receive`

**Issue:**
- MRs that were fully ordered but not received/partially received were not visible in the Report **Requested Items to Order and Receive**
  <img width="1299" alt="Screenshot 2022-02-25 at 12 13 38 PM" src="https://user-images.githubusercontent.com/25857446/155667560-9976d5ea-20c7-4002-aea1-80c57ed7bb3d.png">


**Fix:**
- Remove incorrect query clause that only check if ordered qty < 100
- MR should be visible in report until fully received (cycle complete)
   <img width="1308" alt="Screenshot 2022-02-25 at 12 12 58 PM" src="https://user-images.githubusercontent.com/25857446/155667644-b4aab287-9459-45dd-920c-4f3840691c79.png">

**To test:**
- Create an MR of type **Purchase** for 10 Qty
- Create a PO against this MR for 10 Qty. Check Report
- Create a PR against this PO for 5 Qty (partial). Check Report
- Report must have this MR in it, as it not fully received/purchase cycle is not yet complete.
- Make another PR for remaining 5 Qty, MR must disappear from report
